crypto/src/hash/rescue_prime/utils.rs

```diff
@@ -0,0 +1,75 @@
+use super::Fp;
+use alloc::vec::Vec;
+
+pub fn bytes_to_field_elements(input: &[u8]) -> Vec<Fp> {
+    input
+        .chunks(7)
+        .map(|chunk| {
+            let mut buf = [0u8; 8];
+            buf[..chunk.len()].copy_from_slice(chunk);
+            if chunk.len() < 7 {
+                buf[chunk.len()] = 1;
+            }
+            let value = u64::from_le_bytes(buf);
+            Fp::from(value)
+        })
+        .collect()
+}
```

Comment:
Is this function just for rescue_prime or a function that can be used elsewhere?

Reply:
Currently it 's only use in rescue_prime. Maybe later it can be added to `fields`